README.md

```diff
@@ -1,64 +1,180 @@
+   - In the Redirect URI (optional) section, select **Web** in the combo-box and enter the following redirect URIs: `http://localhost:5000/getAToken`.
+1. Select **Register** to create the application.
+1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. From the app's Overview page, select the **Authentication** section.
+   - In the **Advanced settings** section set **Logout URL** to `http://localhost:5000/logout`
+1. Select **Save**.
+1. From the **Certificates & secrets** page, in the **Client secrets** section, choose **New client secret**:
+
+   - Type a key description (of instance `app secret`),
+   - Select a key duration of either **In 1 year**, **In 2 years**, or **Never Expires**.
+   - When you press the **Add** button, the key value will be displayed, copy, and save the value in a safe location.
+   - You'll need this key later to configure the project in Visual Studio. This key value will not be displayed again, nor retrievable by any other means,
+     so record it as soon as it is visible from the Azure portal.
+1. Select the **API permissions** section
+   - Click the **Add a permission** button and then,
+   - Ensure that the **Microsoft APIs** tab is selected
+   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **User.Read**. Use the search box if necessary.
+   - Select the **Add permissions** button
+
+### Step 3:  Configure the sample to use your Azure AD tenant
+
+In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+#### Configure the pythonwebapp project
+
+> Note: if you used the setup scripts, the changes below will have been applied for you
+
+1. Open the `app_config.py` file
+1. Find the app key `Enter_the_Tenant_Name_Here` and replace the existing value with your Azure AD tenant name.
+1. Find the app key `Enter_the_Client_Secret_Here` and replace the existing value with the key you saved during the creation of the `python-webapp` app, in the Azure portal.
+1. Find the app key `Enter_the_Application_Id_here` and replace the existing value with the application ID (clientId) of the `python-webapp` application copied from the Azure portal.
+
+
+### Step 4: Run the sample
+
+- You will need to install MSAL Python library, Flask framework, Flask-Sessions for server side session management and requests using pip as follows:
+```Shell
+$ pip install msal
+$ pip install flask
+$ pip install Flask-Session
+$ pip install requests
+```
+- If the environment variable for Flask is already set:
+
+Run app.py from shell or command line:
+```Shell
+$ python app.py
+```
+- If the environment variable for Flask is not set:
+
+Type the following commands on shell or command line by navigating to the project directory:
+```Shell
+$ export FLASK_APP=app.py

```

Comment:
I wonder if we don't want to have 2 columns;
- Shell
- MSDos? (where export is replaced by set) ?

Reply:
I tested it just now. Looks like the method mentioned in previous line, `python app.py`, is capable to start the flask server. So I think we can completely skip these environment variable thing, thus also bypass that Shell & MSDos topic.
I'll make such change in the upcoming refactoring commit.
CC: @abhidnya13